lib/helpers.py

```diff
@@ -362,13 +362,50 @@ def formated_time():
     return "{0:04d}-{1:02d}-{2:02d}T{3:02d}:{4:02d}:{5:02d}Z".format(*ct)  # modified to fit the correct format
 
 
-def concat_state_log(machine):
+def log_switcher(log_level: str):
```

Comment:
This name might be misleading/sounds like it has something to do with actually changing the log. "get_log_level_name" or something similar might be a better choice in this case.

Reply:
translate_backend_log_level()